ee/audit/interceptor_ee.go

```diff
@@ -101,6 +116,40 @@ func AuditRequestHttp(next http.Handler) http.Handler {
 	})
 }
 
+func AuditWebSockets(ctx context.Context, req *schema.Request) {
+	if atomic.LoadUint32(&auditEnabled) == 0 {
+		return
+	}
+
+	namespace := uint64(0)
+	var user string
+	if token := req.Header.Get("X-Dgraph-AccessToken"); token != "" {
+		user = getUser(token, false)
+		namespace, _ = x.ExtractNamespaceFromJwt(token)
```

Comment:
The error is ignored here. Do you think that maybe it should be at least logged to WARN to give admins visibility into failed authn attempts in their audits?